ggml/src/ggml-vulkan/ggml-vulkan.cpp

```diff
@@ -299,6 +299,7 @@ struct vk_device_struct {
     vk_pipeline pipeline_cpy_f32_f32, pipeline_cpy_f32_f16, pipeline_cpy_f16_f16;
     vk_pipeline pipeline_contig_cpy_f32_f32, pipeline_contig_cpy_f32_f16, pipeline_contig_cpy_f16_f16;
     vk_pipeline pipeline_cpy_f32_quant[GGML_TYPE_COUNT];
+    vk_pipeline pipeline_cpy_f32_quant_rte[GGML_TYPE_COUNT];
```

Comment:
You can just load the rte variant into the existing pipeline, no need to have variables for both. It's either with rte or without.